templates/home.phtml

```diff
@@ -174,22 +174,22 @@
                 <hr>
             </div>
 
-            <button id="nav-refresh" title="<?= trim(\F3::get('lang_refreshbutton')); ?>"></button>
-            <button id="nav-settings" title="<?= trim(\F3::get('lang_settingsbutton')); ?>"></button>
-            <button id="nav-logout" title="<?= trim(\F3::get('lang_logoutbutton')); ?>"></button>
+            <button id="nav-refresh" title="<?= trim(\F3::get('lang_refreshbutton')); ?>" aria-label="<?= trim(\F3::get('lang_refreshbutton')); ?>"></button>
+            <button id="nav-settings" title="<?= trim(\F3::get('lang_settingsbutton')); ?>" aria-label="<?= trim(\F3::get('lang_settingsbutton')); ?>"></button>
+            <button id="nav-logout" title="<?= trim(\F3::get('lang_logoutbutton')); ?>" aria-label="<?= trim(\F3::get('lang_logoutbutton')); ?>"></button>
 
             <?php if ($this->publicMode === true) : ?>
-            <button id="nav-login" title="<?= trim(\F3::get('lang_loginbutton')); ?>"></button>
+            <button id="nav-login" title="<?= trim(\F3::get('lang_loginbutton')); ?>" aria-label="<?= trim(\F3::get('lang_loginbutton')); ?>"></button>
             <?php endif; ?>
 
-            <a id="nav-copyright" href="https://selfoss.aditu.de" target="_blank" rel="noopener noreferrer">selfoss <?= \F3::get('version'); ?></a>
+            <a id="nav-copyright" href="https://selfoss.aditu.de" target="_blank" rel="noopener noreferrer"><?= trim(\F3::get('html_title')); ?> <?= \F3::get('version'); ?></a>
```

Comment:
I agree with you, but since URL leads the user to the original selfoss website, the name customisation should be fine.
Rest, it is your choice, but I'll request you to include this customisation for the better user experience.

Reply:
> I think this is an information about the version of the software so it should probably always be selfoss, rather than the name administrator chooses for a given installation. Though I would argue that user should not need to care about what program this really is so I would move it to the settings.

Actually, I think for the better user experience, the name should be retrieved from the settings itself.
Also, since URL is unchanged and leads the user to the selfoss homepage, it should be fine. rest, it is your choice.

Reply:
> Also, since URL is unchanged and leads the user to the selfoss homepage, it should be fine.

Well this is precisely my point. I would expect the label to match the target URL so that user knows where will it lead before visiting it.

Also it is followed by a selfoss version number, which, I think, does not make sense when combined with a custom title.

I would just remove this link altogether, as it does not add much value while cluttering the sidebar.

Reply:
> > Also, since URL is unchanged and leads the user to the selfoss homepage, it should be fine.
> 
> Well this is precisely my point. I would expect the label to match the target URL so that user knows where will it lead before visiting it.
> 
> Also it is followed by a selfoss version number, which, I think, does not make sense when combined with a custom title.
> 
> I would just remove this link altogether, as it does not add much value while cluttering the sidebar.

Yeah, removing is also a better option.
It will be great.
Also, when will you commit your changes so that I can pull them and work further?
I also want to introduce a heading level1 on th home page because user always looks for the main heading I.E: h1 for reading about the website/application and for the main content. Since main content is already present, where and with which content we should add h1?